vtex-apps/checkout
# CHUI-10: Add handler for place order button and complete the purchase

#### What problem is this solving?

Adds the `onClick` handler to the "Place Order" button to start the purchase pipeline.

This PR depends on vtex-apps/checkout-payment#9

#### How should this be manually tested?

[Workspace](https://placeorder--checkoutio.myvtex.com/cart/add?sku=289).

You should be able to place the order after correctly filling all the checkout steps on the workspace above.

#### Checklist/Reminders


#### Type of changes


✔️ | Type of Change
---|---
_ | Bug fix 
✔️ | New feature 
_ | Breaking change 
_ | Technical improvements
